<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1032</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>931</width>
      <height>561</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QGroupBox {
 
 background: qradialgradient(cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4, radius: 1.35, stop: 0 #fff, stop: 1 #888) ;/*background: linear-gradient(rgba(0,0,0,.5), rgba(0,0,0,.5)),url(feuille.jpg); 
    background-position: center;
    background-size: cover;
    display: flex;
    justify-content: center;
    align-items: center;
    font-family: 'Poppins', sans-serif*/
 
 color: rgb(255, 255, 255);
  border: 1px solid #ffffff;

    margin-top: 1ex; /* leave space at the top for the title */
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top center; /* position at the top center */
    padding: 0 10px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF0ECE, stop: 1 #FFFFFF);
}


QPushButton {
border: 2px solid gray;
border-radius: 10px;
padding: 0 8px;
background-color:blue
}

 QPushButton:hover:!pressed{ background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF0ECE, stop: 1 #FFFFFF); } 
QPushButton:hover { background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF0ECE, stop: 1 #FFFFFF); }</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="Home">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="title">
         <string>Smart Printing System</string>
        </property>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>100</y>
           <width>491</width>
           <height>61</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Lucida Console</family>
           <pointsize>13</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(85, 170, 255);</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; text-decoration: underline;&quot;&gt;Bienvenue,vous êtes le livreur!!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QPushButton" name="livrButton">
         <property name="geometry">
          <rect>
           <x>360</x>
           <y>240</y>
           <width>141</width>
           <height>61</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(85, 170, 255);</string>
         </property>
         <property name="text">
          <string>Livraisons</string>
         </property>
        </widget>
        <widget class="QPushButton" name="deconnecterButton">
         <property name="geometry">
          <rect>
           <x>780</x>
           <y>470</y>
           <width>121</width>
           <height>61</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">
background-color: rgb(255, 255, 127);</string>
         </property>
         <property name="text">
          <string>Déconnecter</string>
         </property>
        </widget>
        <widget class="QTextBrowser" name="textBrowser_4">
         <property name="geometry">
          <rect>
           <x>-10</x>
           <y>420</y>
           <width>131</width>
           <height>121</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(:/img/Projet qt/delivery.png);</string>
         </property>
        </widget>
        <zorder>textBrowser_4</zorder>
        <zorder>label_2</zorder>
        <zorder>livrButton</zorder>
        <zorder>deconnecterButton</zorder>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="livraisons">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="title">
         <string/>
        </property>
        <widget class="QTabWidget" name="historique">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>10</y>
           <width>911</width>
           <height>541</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>911</width>
           <height>0</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="currentIndex">
          <number>2</number>
         </property>
         <widget class="QWidget" name="tab_4">
          <attribute name="title">
           <string>Ajouter</string>
          </attribute>
          <widget class="QPushButton" name="homeButton_4">
           <property name="geometry">
            <rect>
             <x>780</x>
             <y>440</y>
             <width>111</width>
             <height>51</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 127);
</string>
           </property>
           <property name="text">
            <string>Home</string>
           </property>
          </widget>
          <widget class="QSplitter" name="splitter_3">
           <property name="geometry">
            <rect>
             <x>280</x>
             <y>30</y>
             <width>341</width>
             <height>411</height>
            </rect>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QLabel" name="label">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(85, 170, 255);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; text-decoration: underline;&quot;&gt;Ajouter une livraison :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
           <widget class="QWidget" name="layoutWidget">
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Id:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="le_id"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Date:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="la_date"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_17">
               <property name="text">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Frais:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="le_frais"/>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_18">
               <property name="text">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Localisation:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLineEdit" name="l_adress"/>
             </item>
            </layout>
           </widget>
           <widget class="QPushButton" name="ajouter">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(85, 170, 255);</string>
            </property>
            <property name="text">
             <string>Ajouter</string>
            </property>
           </widget>
          </widget>
          <widget class="QTextBrowser" name="textBrowser_2">
           <property name="geometry">
            <rect>
             <x>-10</x>
             <y>380</y>
             <width>131</width>
             <height>121</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">border-image: url(:/img/Projet qt/delivery.png);</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="tab_5">
          <attribute name="title">
           <string>Afficher</string>
          </attribute>
          <widget class="QTableView" name="table_livraison">
           <property name="geometry">
            <rect>
             <x>120</x>
             <y>280</y>
             <width>651</width>
             <height>211</height>
            </rect>
           </property>
          </widget>
          <widget class="QTextBrowser" name="textBrowser_3">
           <property name="geometry">
            <rect>
             <x>-10</x>
             <y>380</y>
             <width>131</width>
             <height>121</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">border-image: url(:/img/Projet qt/delivery.png);</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="recherche">
           <property name="geometry">
            <rect>
             <x>350</x>
             <y>59</y>
             <width>181</width>
             <height>31</height>
            </rect>
           </property>
          </widget>
          <widget class="QComboBox" name="comboBox_tri">
           <property name="geometry">
            <rect>
             <x>580</x>
             <y>120</y>
             <width>192</width>
             <height>41</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">border-color: rgb(170, 170, 255);
selection-background-color: rgb(170, 170, 255);


font: 75 10pt &quot;MS Shell Dlg 2&quot;;
</string>
           </property>
           <item>
            <property name="text">
             <string>            TRIER</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Tri par frais</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Tri par id croissant</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Tri par id Descendant</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Tri par localisation</string>
            </property>
           </item>
          </widget>
          <widget class="QLabel" name="label_3">
           <property name="geometry">
            <rect>
             <x>350</x>
             <y>20</y>
             <width>181</width>
             <height>41</height>
            </rect>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; text-decoration: underline; color:#ffaaff;&quot;&gt;Rechercher:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
          <widget class="QPushButton" name="homeButton_5">
           <property name="geometry">
            <rect>
             <x>780</x>
             <y>440</y>
             <width>111</width>
             <height>51</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 127);
</string>
           </property>
           <property name="text">
            <string>Home</string>
           </property>
          </widget>
          <widget class="QPushButton" name="supprimer2">
           <property name="geometry">
            <rect>
             <x>350</x>
             <y>194</y>
             <width>181</width>
             <height>51</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(85, 255, 127);
selection-color: rgb(0, 85, 0);</string>
           </property>
           <property name="text">
            <string>Supprimer</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_3">
           <property name="geometry">
            <rect>
             <x>780</x>
             <y>370</y>
             <width>111</width>
             <height>61</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(85, 255, 255);</string>
           </property>
           <property name="text">
            <string>Accès Site Web</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="le_id_sup_2">
           <property name="geometry">
            <rect>
             <x>350</x>
             <y>154</y>
             <width>181</width>
             <height>31</height>
            </rect>
           </property>
           <property name="focusPolicy">
            <enum>Qt::StrongFocus</enum>
           </property>
          </widget>
          <widget class="QLabel" name="label_22">
           <property name="geometry">
            <rect>
             <x>350</x>
             <y>114</y>
             <width>181</width>
             <height>31</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(85, 0, 255);

font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; text-decoration: underline; color:#55ff7f;&quot;&gt;Id a supprimer:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_4">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>10</y>
             <width>301</width>
             <height>31</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(85, 170, 255);</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; text-decoration: underline;&quot;&gt;Veuillez entrer les modifications:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_2">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>210</y>
             <width>251</width>
             <height>51</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(85, 170, 255);</string>
           </property>
           <property name="text">
            <string>Modifier</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_19">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>130</y>
             <width>51</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;Frais:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="la_date_2">
           <property name="geometry">
            <rect>
             <x>130</x>
             <y>90</y>
             <width>161</width>
             <height>31</height>
            </rect>
           </property>
          </widget>
          <widget class="QLineEdit" name="le_id_2">
           <property name="geometry">
            <rect>
             <x>130</x>
             <y>50</y>
             <width>161</width>
             <height>31</height>
            </rect>
           </property>
          </widget>
          <widget class="QLineEdit" name="le_frais_2">
           <property name="geometry">
            <rect>
             <x>130</x>
             <y>130</y>
             <width>161</width>
             <height>31</height>
            </rect>
           </property>
          </widget>
          <widget class="QLineEdit" name="l_adress_2">
           <property name="geometry">
            <rect>
             <x>130</x>
             <y>170</y>
             <width>161</width>
             <height>31</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="label_15">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>90</y>
             <width>51</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;Date:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_20">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>170</y>
             <width>121</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;Localisation:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_16">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>50</y>
             <width>31</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;Id:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton">
           <property name="geometry">
            <rect>
             <x>780</x>
             <y>300</y>
             <width>111</width>
             <height>61</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(170, 170, 127);</string>
           </property>
           <property name="text">
            <string>Actualiser</string>
           </property>
          </widget>
          <zorder>pushButton_2</zorder>
          <zorder>label_19</zorder>
          <zorder>la_date_2</zorder>
          <zorder>le_id_2</zorder>
          <zorder>le_frais_2</zorder>
          <zorder>l_adress_2</zorder>
          <zorder>label_15</zorder>
          <zorder>label_20</zorder>
          <zorder>label_16</zorder>
          <zorder>textBrowser_3</zorder>
          <zorder>table_livraison</zorder>
          <zorder>recherche</zorder>
          <zorder>comboBox_tri</zorder>
          <zorder>label_3</zorder>
          <zorder>homeButton_5</zorder>
          <zorder>supprimer2</zorder>
          <zorder>pushButton_3</zorder>
          <zorder>le_id_sup_2</zorder>
          <zorder>label_22</zorder>
          <zorder>label_4</zorder>
          <zorder>pushButton</zorder>
         </widget>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>Historique</string>
          </attribute>
          <widget class="QTextBrowser" name="textBrowser_5">
           <property name="geometry">
            <rect>
             <x>-10</x>
             <y>380</y>
             <width>131</width>
             <height>121</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">border-image: url(:/img/Projet qt/delivery.png);</string>
           </property>
          </widget>
          <widget class="QTableView" name="tablehistorique">
           <property name="geometry">
            <rect>
             <x>170</x>
             <y>120</y>
             <width>521</width>
             <height>291</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="label_5">
           <property name="geometry">
            <rect>
             <x>250</x>
             <y>60</y>
             <width>381</width>
             <height>41</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(170, 0, 127);</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; text-decoration: underline;&quot;&gt;Liste des modifications récentes:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_4">
           <property name="geometry">
            <rect>
             <x>380</x>
             <y>420</y>
             <width>111</width>
             <height>61</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(170, 0, 127);</string>
           </property>
           <property name="text">
            <string>Actualiser</string>
           </property>
          </widget>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1032</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew_User">
   <property name="text">
    <string>New User</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
